software/mk/Makefile.tail_rules

```diff
@@ -34,7 +34,7 @@ spike_call_log: pp_spike_trace
 CLEAN_ITEMS ?=
 
 clean:
-	-rm -rf *.o *.a *.jou *.log *.pelog *.pb bsg_manycore_io_complex_rom.v *.riscv *.wdb *.bin *.hex
+	-rm -rf *.o *.a *.jou *.log *.pelog *.pb bsg_manycore_io_complex_rom.v *.riscv *.wdb *.bin *.hex *.ld
```

Comment:
If .ld files are now removed by clean, I think you should add .ld files to .gitignore

Reply:
Or at least software/spmd/.gitignore

Reply:
Sounds good!